content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: blog
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-03-22
```

Comment:
Typically, #sig-docs-blog folks + release team, will assign a date for this closer to or soon after the release is GA, let's make sure to change to that here and in the markdown file name once we know that date.

Reply:
btw I asked there https://kubernetes.slack.com/archives/CNT9Y603D/p1679485223404749